### PR TITLE
bugfix([DST-627]): replace useState import from storybook

### PR DESCRIPTION
**Pull request by aromko**
# Reviewers:
@marigold-ui/developer

### CODE REVIEW

File: .changeset/new-paws-carry.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+---
+"@marigold/components": patch
+---
+
+bugfix([DST-627]): replace useState import from storybook
+
+Some controlled stories in Storybook were incorrect because the useState import from react was used, resulting in an error.
+
+The following components were affected:
```
Comment (aromko) [inline .changeset/new-paws-carry.md:9]:
```suggestion
The stories of the following component were affected:
```

**Comment by sebald** [comment]
@aromko I think we had this issues a few times. Is it maybe also related to this: https://stackoverflow.com/questions/71993857/using-usestate-along-with-global-decorators-throws-error-storybook-preview-hooks/77859321#77859321

**Comment by aromko** [comment]
> @aromko I think we had this issues a few times. Is it maybe also related to this: https://stackoverflow.com/questions/71993857/using-usestate-along-with-global-decorators-throws-error-storybook-preview-hooks/77859321#77859321

Yeah. And I looked now through all stories. So it should be fixed now.

**Comment by sebald** [comment]
@aromko but using their preview API seems not the correct way to solve this. Even in their official docs, they don't use this API:

https://storybook.js.org/docs/8.5/writing-stories#working-with-react-hooks

**Comment by aromko** [comment]
> @aromko I think we had this issues a few times. Is it maybe also related to this: https://stackoverflow.com/questions/71993857/using-usestate-along-with-global-decorators-throws-error-storybook-preview-hooks/77859321#77859321

To put it to the end doesn't work. And i go though some issues on their github repo. It seems to came up multiple times (latest https://github.com/storybookjs/storybook/issues/29189). Another solution was to use Story() instead of <Story /> and we do this already. So @sebald have another solution in mind or let it for now?

**Comment by sebald** [comment]
@aromko so changing the order didn't fix it?

**Comment by aromko** [comment]
> @aromko so changing the order didn't fix it?

If I haven't done something wrong - it doesn't helped

**Review by sebald — APPROVED** [review]

**Pull request merged**